source/services/MonitoringService.ts

```diff
@@ -0,0 +1,32 @@
+import { ComponentType } from "react";
+import env from "react-native-config";
+
+import SentryMonitoringService from "./SentryMonitoringService";
+
+export interface MonitoringService {
+  init: () => void;
+  sendError: (Error: Error | string | null) => void;
+  wrap: (component: ComponentType<unknown>) => ComponentType<unknown>;
+}
+
+const nullMonitoringService: MonitoringService = {
+  init: () => undefined,
+  sendError: () => undefined,
+  wrap: (component) => component,
+};
+
+class MonitoringServiceFactory {
+  static getMonitoringService(environment: string): MonitoringService {
+    if (environment === "production") {
```

Comment:
Do we have constants for `"production"` and `"development"`?

Reply:
Doesn't seems like it